### ISSUE TITLE
Attempt At Reading

### ISSUE THREAD

**Issue by Si13x** [robertdavidgraham/cybersymposium#2]
Hi,

I grabbed a copy of this and took a crack at figuring out a snippet to see if it was obviously bogus.  I picked Target_MachineID_HEX.rtf and grabbed a chunk out of it as my target as it seemed likely to me that if the content were real, it would be simple and number heavy, and even my rusty Chinese language skills would be able to read something, or it'd be text a web translator could handle.

A first pass at it and it appears to absolute rubbish -- Chinese language Lorem Ipsum.  More specifically, someone took piles of random Chinese-language advertising text (ads for shoes,etc), copied and pasted it repeatedly into a .rtf, then converted that to hex to try to tart it up into something dramatic looking.

I found some of the exact chunks of Chinese text referenced in other, innocuous places around the web, including in a tool for scraping Chinese ads.

https://macosx.com/threads/has-my-system-been-compromised-chinese-characters-anydesk.324034/

https://github.com/cs50/adminer50

Hope you find this useful or at least entertaining while you wait for Lindell to show you his "big reveal"...

-Best,
Silex
[Sample Conversion_Target_MachineID_HEX.txt](https://github.com/robertdavidgraham/cybersymposium/files/6968194/Sample.Conversion_Target_MachineID_HEX.txt)

**Comment by everythingscomingupmilhouse** [comment]
Hi, I've been trying to reproduce this myself, but I can't get anything out of it really. Exactly how did you do the hex->UTF-16 conversion? I've converted the hex to bytes, clipped off the rtf header, and I've got a lot of characters escaped with `\'`. I've tried processing it a couple different ways but I can't get very far without python telling me about an invalid utf-16 surrogate or whatever. 🤪

**Comment by Si13x** [comment]
Hi Milhouse,

I took the easy route on this one as I hate dealing with both hex and mystery Asian language encodings, and used Dencode (set to UTF16) to caveman-smash my way to output that I could quickly sniff-test.

Reasoning Braindump:

Given the readable RTF header indicating a US English computer being involved (ansicpg1252 encoding), and what I already knew about Lindell and his source, Montgomery, I was betting that the contents of these files would be Lorem Ipsum-type content scraped from some random corner of the web, if they were recoverable at all.  Being unrecoverable wreckage I considered very likely, either as an intentional attempt at obfuscating the contents and further stalling actually giving anything real to Graham and the other guests, or out of sheer ignorance of a scammer who doesn't know about the stupid gotchas of managing files with non-English content.

Alternatively, if they were by some miracle real (or a less obvious hoax), as I mentioned earlier, they'd be short, simple strings I might be able to actually read.

I played with a few different encodings but focused on UTF16 due to Graham's comments and also due to it being popular these days for Simplified Mainland Chinese character sets.  I was figuring Montgomery would expend at least enough effort to look up the Chinese govt's preferred encoding.

TL;DR -- I figured this would be garbage and lazily went for the low-hanging fruit.  I could be totally off base, of course, this stuff isn't my area of expertise.

Dencode Stuff:

Dencode source:  https://github.com/mozq/dencode-web

Web frontend to the Hex string tools: https://dencode.com/string/hex
